enterprise/app/tap/tap.tsx

```diff
@@ -394,18 +396,54 @@ export default class TapComponent extends React.Component<Props, State> {
           <div className="container narrow">
             <div className="empty-state history">
               <h2>No CI tests found in the last week!</h2>
-              <p>
-                Seems like you haven't done any builds marked as CI. Add <b>--build_metadata=ROLE=CI</b> to your builds
-                to see a CI test grid. You'll likely also want to provide a commit SHA and optionally a git repo url.
-                <br />
-                <br />
-                Check out the Build Metadata Guide below for more information on configuring these.
-                <br />
-                <br />
-                <a className="button" target="_blank" href="https://buildbuddy.io/docs/guide-metadata">
-                  View the Build Metadata Guide
-                </a>
-              </p>
+              {this.isV2 ? (
+                <p>
+                  To see a CI test grid, make sure your CI tests are configured as follows:
+                  <ul>
+                    <li>
+                      Add <code className="inline-code">--build_metadata=ROLE=CI</code> to your CI bazel test command
+                    </li>
+                    <li>
+                      Add a file called <code className="inline-code">workspace_status.sh</code> to the root of your
```

Comment:
Not sure how I feel about being so explicit about this. Another totally reasonable approach is to use something like Buildkite where all of the environment variables are already set.

Reply:
Ah right. Replaced these with links to the build metadata guide